docs/docs/widgets/color-picker.md

```diff
@@ -26,19 +26,17 @@ Invalid Color : #0000, "black" , rgb(0,0,0) ,
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Events
 
-To add an event to a color-picker component, click on the widget handle to open the widget properties on the right sidebar. Go to the **Events** section and click on **+ Add handler**.
-
 | <div style={{ width:"100px"}}> Event </div> | <div style={{ width:"100px"}}> Description </div> |
 |:----------- |:----------- |
-| On change | On change event is triggered when the color is changed on the color-picker|
+| On change | Triggers whenever the user types something in the text input.|
```

Comment:
Hey @NohaFahmi, you were not supposed to exactly copy the content from the reference but instead, you were expected to reframe this content according to the color picker component, please update this.